app/controllers/physical_ticket_controller.rb

```diff
@@ -0,0 +1,29 @@
+class PhysicalTicketController < ApplicationController
```

Comment:
if we create Physical Tickets in ticket purchase controller then basically we are creating tickets before their payment is made through stripe.